web/src/widget/text_input.rs

```diff
@@ -67,7 +75,13 @@ impl<'a, Message> TextInput<'a, Message> {
     {
         Self {
             _state: state,
-            placeholder: String::from(placeholder),
+            placeholder: if let Some(p) = placeholder {
```

Comment:
i'm not sure if there is a better way to do this. it seems like there should be, but i don't know enough about dodrio

Reply:
placeholder.map(Into::into).unwrap_or_default() could work